api/controllers/topics-controllers.js

```diff
@@ -9,9 +9,8 @@ exports.getTopics = (req,res,next) => {
     .catch(next)
 }
 
-exports.getEndpoints = (req,res,next) => {
+exports.getEndpoints = (req,res) => {
     res.status(200).send({allEndPoints})
-    .catch(next)
```

Comment:
As you're sending back a pre-built JSON object, I suppose a catch block isn't strictly necessary here